### PR TITLE
Adding yt warning to examples notebooks

### PR DESCRIPTION
**Pull request by opotowsky**
This addresses a comment in issue #1276 about warning users about the broken yt problem. The notebooks are not rerun, and can't be until the yt changes are made. Also fixes urllib to requests where necessary.

### CODE REVIEW

**Review by kkiesling — APPROVED** [review]

File: CHANGELOG.rst [hunk 1]
```diff
@@ -87,6 +87,7 @@ v0.7.0 RC2
    * in Chainsolve Transmutation, rerun example to update output
    * pytaps to pymoab update in intro tutorial (#1282)
    * in material_library example, updated import and class usage to reflect MaterialLibrary changes (#1280)
+   * adds broken yt warning to examples notebooks, updates deprecated urlretrieve to requests (#1295)
```
Comment (kkiesling) [inline CHANGELOG.rst:92]: should this be two separate change log entries?

Comment (opotowsky) [reply CHANGELOG.rst:92]: everyone is making the urlretrieve changes on their own w indiviual notebooks,so I don't know!

Comment (kkiesling) [reply CHANGELOG.rst:92]: ah- nbd it's probably fine to leave as is. These two statements separated by a comma are just seeming unrelated which is why I suggest two different entries.

**Comment by gonuke** [comment]
thanks @opotowsky

**Pull request merged**